salt/modules/cmdmod.py

```diff
@@ -2653,16 +2654,25 @@ def shells():
     return ret
 
 
-def shell_info(shell):
+def shell_info(shell, **kwargs):
```

Comment:
We do not typically accept `**kwargs` in execution modules. Why is this necessary here?